src/frontend/components/StudentLiveWrapper/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+
+import DashboardVideoLiveJitsi from 'components/DashboardVideoLiveJitsi';
+import { LiveVideoPanel } from 'components/LiveVideoPanel';
+import { StudentLiveControlBar } from 'components/StudentLiveControlBar';
+import { StudentLiveInfoBar } from 'components/StudentLiveInfoBar';
+import VideoPlayer from 'components/VideoPlayer';
+import {
+  LivePanelItem,
+  useLivePanelState,
+} from 'data/stores/useLivePanelState';
+import { useVideo } from 'data/stores/useVideo';
+import { Video } from 'types/tracks';
+
+import { StudentLiveLayout } from './StudentLiveLayout';
+
+interface LiveStreamerConfiguration {
+  type: 'on_stage';
+}
+
+interface LiveViewerConfiguration {
+  type: 'viewer';
+  playerType: string;
+}
```

Comment:
Both type can be defined with an enum and you can then use this enum later in the code to test the configuration type.